core/storage/app_feedback_report/gae_models.py

```diff
@@ -284,6 +306,7 @@ def get_export_policy(cls):
 
     @classmethod
     def export_data(cls, user_id):
+        # type: (Text) -> Dict[Text, Any]
```

Comment:
Can we replace the `Any` here?

Reply:
+1, I also have this question.

@hardikkat24 we've talked before about not using Any. All such usages should be exceptions and need full justification via the doc approach we discussed before, otherwise they'll just be rejected at PR review time.

Reply:
@seanlip  Added the explanation in the doc.
One more thing, there are some error which I'm not able to fix nor am I able to find solutions online or on the public channels, so the last option which I see is to ignore errors in these lines. So should I also add them to the doc? Thanks!

Reply:
Yes. The requirement is to not use Any/ignore, and any exceptions to that should be discussed in the doc before they make it into a PR.

Reply:
And one more thing, most used ignores will be of:
1. no-untyped-calls due to calling an untyped function
2. arg-type in test files: passing wrong arguments in test files deliberately.